core/src/main/java/org/axonframework/eventhandling/SimpleEventHandlerInvoker.java

```diff
@@ -37,6 +38,7 @@
  */
 public class SimpleEventHandlerInvoker implements EventHandlerInvoker {
 
+    private final List<?> eventListenerDelegates;
```

Comment:
Why introduce a `eventListenerDelegates` if it is _always_ instantiated with the `eventListeners`? Just use the `eventListeners` instead then.

Reply:
`eventListeners` are a list of `EventListeners`, and these are the actual delegates (the one which to the handling of an event). I agree that we should come up with better naming, so I'll spend some time on it.

Reply:
Great, thanks for that!